importexport/musicxml/importmxmlpass2.cpp

```diff
@@ -5432,6 +5432,9 @@ void MusicXMLParserNotations::slur()
       if (notation.attribute("type") == "stop") {
             _slurStop = true;
             }
+      if (notation.attribute("type") == "start") {
```

Comment:
not an `else if` here?

Reply:
Good call; I'll go ahead and make that change!